crates/holochain/src/core/sys_validate.rs

```diff
@@ -1,10 +1,10 @@
 //! # System Validation Checks
 //! This module contains all the checks we run for sys validation
+#![allow(deprecated)]
```

Comment:
Any thoughts on just putting this at the top of our lib.rs so we could actually make use of deprecation more officially?